go/kbfs/libkbfs/folder_branch_ops.go

```diff
@@ -499,13 +499,20 @@ func newFolderBranchOps(
 	fbo.fbm = newFolderBlockManager(appStateUpdater, config, fb, bType, fbo)
 	fbo.rekeyFSM = NewRekeyFSM(fbo)
 	if config.DoBackgroundFlushes() && bType == standard {
-		fbo.doneWg.Add(1)
-		go fbo.backgroundFlusher()
+		fbo.goTracked(fbo.backgroundFlusher)
 	}
 
 	return fbo
 }
 
+func (fbo *folderBranchOps) goTracked(f func()) {
```

Comment:
It sucks that this pollutes the stack a bunch, but I don't have any suggestions on reducing that; it's definitely cleaner to do it this way than the other way.